From 90b6fcf23a926806c1d1d5b18e1945c6edbcb022 Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@foss.st.com>
Date: Thu, 29 Aug 2024 11:39:03 +0200
Subject: [PATCH] arm64: dts: st: add assert/deassert delay nRST of PHY on
 stm32mp215f-dk

LAN8742 is used on MP215F-DK and delay of nRST assert/deassert
are recommended.

Signed-off-by: Christophe Roullier <christophe.roullier@foss.st.com>
Change-Id: I48fabe1173f40d7cd576556c23348dfad423c178
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/400876
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp215f-dk.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp215f-dk.dts b/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
index 9452ef27961b..ed758c07833c 100644
--- a/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
@@ -209,6 +209,8 @@ mdio1 {
 		phy0_eth1: ethernet-phy@0 {
 			compatible = "ethernet-phy-id0007.c131";
 			reset-gpios =  <&gpioh 11 GPIO_ACTIVE_LOW>;
+			reset-assert-us = <1000>;
+			reset-deassert-us = <30000>;
 			reg = <0>;
 		};
 	};
-- 
2.39.5

